src/components/track-tile/TrackTile.tsx

```diff
@@ -109,70 +218,65 @@ export const TrackTile = ({
     }
   }, [onLoad, isLoaded, index, opacity])
 
+  if (is_delete || user?.is_deactivated) {
```

Comment:
maybe we do these checks in the parent component?

Reply:
Love it!